libraries/integer/java/Utils.java

```diff
@@ -59,6 +59,10 @@ private static BigInteger toUnsignedBigInteger(long i) {
         }
     }
 
+    public static BigInteger toUnsignedBigInteger(int i) {
+      return Utils.toUnsignedBigInteger(Long.valueOf(i));
```

Comment:
Very subtle bug here: if you pass in negative integers, you'll get negative BigIntegers which is not the intended behaviour. Use `toUnsignedLong` instead of `Long.valueOf`.